docs/Public_API.rst

```diff
@@ -133,7 +133,7 @@ Application Metric API
 Route
 '''''
 
-    GET /v1/apps/8d0cee08-23ad-4813-a779-ad8118ea0b91/metric_histories/memoryused
+    GET /v1/apps/:guid/metric_histories/memoryused
 
 Parameters
 ''''''''''
```

Comment:
For the **Parameters** definition:
1. metric-type:
    - `metric_type` with '_' instead of '-' is used in `GET /v1/apps/:guid/metric_histories/:metric_type`
    - need we add custom metric in the description?
2. order-direction: `The scaling history will be order by timestamp asc or desc.` -> `The metric history will be order by timestamp asc or desc.`

Reply:
@zyjiaobj 
1 "metric_type" is in path so it is different from query parameters, such as "start-time" or "order-direction".
2 "be order" or "be ordered"?

Reply:
I mean "scaling history" should be change to "metric history", as this is `metric_histories` api.
Also, "be ordered" should be better.